packages/types/package.json

```diff
@@ -34,9 +34,10 @@
     "githook:precommit": "lint-staged && pnpm run types"
   },
   "dependencies": {
+    "@cryptkeeperzk/rln-proof": "workspace:^",
     "@semaphore-protocol/proof": "^3.10.1",
     "@zk-kit/incremental-merkle-tree": "^1.1.0",
-    "rlnjs": "^2.0.8"
+    "discreetly-interfaces": "^0.1.13"
```

Comment:
Why do we need this dependency?

Reply:
We needed it for those two usages: 
- `export type { MessageI as IRLNMessage, RoomI as IRLNRoom } from "discreetly-interfaces";`
https://github.com/CryptKeeperZK/crypt-keeper-extension/blob/ce40ae4f62ffd5ccc4f51c0b629e021177a79773/packages/types/src/proof/rlnProof.ts#L6

- `import { str2BigInt } from "discreetly-interfaces";`
https://github.com/CryptKeeperZK/crypt-keeper-extension/blob/ce40ae4f62ffd5ccc4f51c0b629e021177a79773/packages/zk/src/proof/protocols/utils.ts#L6

Reply:
Actually I uninstalled it. I just figured out that we wont need RoomI or MessageI for RLN.